components/terraform/Header.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="hero header row justify-content-center">
+        <div class="col-12 col-md-12 col-lg-10">
+            <div class="row">
+                <div class="col-12 col-md-6">
+                    <h1>Kestra Terraform Provider: Deploy All Your Workflows in a CI/CD Process</h1>
+                    <p>Manage and deploy Kestra workflows directly within your existing Terraform environment, eliminating the need for separate configurations.</p>
```

Comment:
```suggestion
                    <h1>Kestra Terraform Provider: Deploy All Your Workflows <br/> in a CI/CD Process</h1>
                    <p>Manage and deploy Kestra workflows directly within your existing Terraform environment, eliminating the need for separate configurations.</p>
```
there must be a line break before "in a CI/CD Process"